Civi/Api4/Generic/AutocompleteAction.php

```diff
@@ -132,6 +132,28 @@ public function _run(Result $result) {
 
     $entityName = $this->getEntityName();
 
+    // Get default display from system settings
+    if (!$this->display) {
+      try {
+        $displaySettings = \Civi::settings()->get('autocomplete_displays');
+        $this->display = $displaySettings[$entityName] ?? NULL;
+        if ($this->display) {
+          $this->display = \Civi\Api4\SearchDisplay::get(FALSE)
+            ->setSelect(['*', 'type:name'])
+            ->addWhere('name', '=', $this->display)
+            ->addWhere('type', '=', 'autocomplete')
+            ->execute()->single();
```

Comment:
I think you might be better to do `first()` & only continue if it has not been deleted without updating the setting

Reply:
@eileenmcnaughton that's the idea with wrapping this in try/catch